EssentialCSharp.Web.Tests/Integration/CaptchaTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using System.Net;
+using EssentialCSharp.Web.Services;
+using Microsoft.AspNetCore.Mvc.Testing;
+
+namespace EssentialCSharp.Web.Extensions.Tests.Integration;
+
+public class CaptchaTests : IClassFixture<WebApplicationFactory<Program>>
+{
+    private readonly WebApplicationFactory<Program> _factory;
+
+    public CaptchaTests(WebApplicationFactory<Program> factory)
+    {
+        _factory = factory;
+    }
+
+    [Fact]
+    public async Task CaptchaService_Verify_Success()
+    {
+        ICaptchaService? _CaptchaService = _factory.Services.GetService<ICaptchaService>();
+        Assert.NotNull(_CaptchaService);

```

Comment:
nit: I would just use `GetRequiredService` which does the null check for you.
```suggestion
        ICaptchaService captchaService = _factory.Services.GetRequiredService<ICaptchaService>();

```